geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/process/stats/StatsIteratorProcess.scala

```diff
@@ -75,12 +75,13 @@ class StatsVisitor(features: SimpleFeatureCollection, statString: String, encode
   import scala.collection.JavaConversions._
   val origSft = features.getSchema
 
-  lazy val transformSF: TransformSimpleFeature = TransformSimpleFeature(origSft, properties)
+  lazy val transformSFT = QueryPlanner.queryToTransformSFT(origSft, properties)
+  lazy val transformSF: TransformSimpleFeature = TransformSimpleFeature(origSft, transformSFT, properties.mkString(","))
```

Comment:
i think you will need to get the transform string here instead of `properties.mkString`